tests/telemetry/test_telemetry.py

```diff
@@ -133,3 +137,47 @@ def test_telemetry_ouput(duthosts, rand_one_dut_hostname, ptfhost, localhost):
     result = str(show_gnmi_out)
     inerrors_match = re.search("SAI_PORT_STAT_IF_IN_ERRORS", result)
     pytest_assert(inerrors_match is not None, "SAI_PORT_STAT_IF_IN_ERRORS not found in gnmi_output")
+
+def test_sysuptime(duthosts, rand_one_dut_hostname, ptfhost, setup_streaming_telemetry, localhost):
+    """
+    @summary: Run pyclient from ptfdocker and test the dataset 'system uptime' to check
+              whether the value of 'system uptime' was float number and whether the value was
+              updated correctly.
+    """
+    logger.info("start test the dataset 'system uptime'")
+    duthost = duthosts[rand_one_dut_hostname]
+    dut_ip = duthost.mgmt_ip
+    cmd = 'python /gnxi/gnmi_cli_py/py_gnmicli.py -g -t {0} -p {1} -m get -x proc/uptime -xt OTHERS \
+           -o "ndastreamingservertest"'.format(dut_ip, TELEMETRY_PORT)
+    system_uptime_info = ptfhost.shell(cmd)["stdout_lines"]
+    system_uptime_1st = 0
+    found_system_uptime_field = False
+    for line_info in system_uptime_info:
+        if "total" in line_info:
+            try:
+                system_uptime_1st = float(line_info.split(":")[1].strip())
```

Comment:
Why do we need float comparison? We just marshal and send value irrespective of datatype?

Reply:
This is not `float` cast and it was used to convert a string into a `float` number in python. This is the pytest script which aims to verify whether the value of `system uptime` returned by the `gnmi_cli` command was a valid number, exactly speaking, it is a `float` number. If you review this link: https://github.com/c9s/goprocinfo/blob/master/linux/uptime.go, the type of `total` field in `uptime` structure is a `float64` in Go.